webapp/src/components/modals/create_issue/create_issue.jsx

```diff
@@ -61,6 +61,7 @@ export default class CreateIssueModal extends PureComponent {
         'priority',
         'description',
         'summary',
+        'reporter',
```

Comment:
I thought we were not going to show it?

Reply:
So I am still confused about it. I did not see it shown, anywhere. If I did not add it to the list, the webapp wouldn't even submit without it.